### ISSUE TITLE
Cannot find module jvm-mon.js on Ubuntu 16.04

### ISSUE THREAD

**Issue by Kaijiro** [ajermakovics/jvm-mon#13]
Hey,

I downloaded your tar, extracted it, did the configuration for Java, and when I executed jvm-mon, here is the error I got : 

```
OpenJDK 64-Bit Server VM warning: You have loaded library /home/tristan/libj2v8_linux_x86_64.so which might have disabled stack guard. The VM will try to fix the stack guard now.
It's highly recommended that you fix the library with 'execstack -c <libfile>', or link it with '-z noexecstack'.

node.js:376
        callback();
        ^
module.js:341: Error: Cannot find module '/home/tristan/Téléchargements/jvm-mon-0.1/bin/src/dist/jvm-mon.js'
```

**Comment by ajermakovics** [comment]
Hi @Kaijiro 

Was that by running from inside the extracted `jvm-mon-0.1` directory or from another location? Sounds similar to #8 and it should be fixed in v0.2 which was just released

**Comment by Kaijiro** [comment]
Ok, so I tried both because I didn't know anymore what I tried. 
Executing `bin/jvm-mom` from the extracted directory works perfectly.
Executing it from somewhere doesn't work. I should have searched before posting, sorry :/

Thank you for your feedback.
